job-distributor/v1/job/job.proto

```diff
@@ -149,7 +149,10 @@ message ProposeJobResponse {
 message RevokeJobRequest {
   oneof id_oneof {
     string id = 1; // Unique identifier of the proposal to revoke.
-    string uuid = 2; // Universally unique identifier of the proposal to revoke.
+    // Universally unique identifier of the proposal to revoke.
+    // This will not work when the same job uuid is proposed for multiple different nodes.
+    // Deprecated: Use `id` instead.
+    string uuid = 2 [deprecated = true];
```

Comment:
I think you also need to deprecate in DeleteJobRequest, UpdateJobRequest and GetJobRequest right?

Reply:
oh man, how did miss this, in my mind i thought they all share the same haha. Let me add them now